### DISCUSSION TITLE
Animate Grid View

### DISCUSSION THREAD

**Discussion by unwize** [Textualize/textual#423] [Q&A]
Hello all. I have been trying to animate a `GridView`-Like widget, but am having trouble. I'd like to animate it moving on and off the screen [like in this example.](https://github.com/Textualize/textual/blob/main/examples/animation.py)

It isn't possible to actually call `animate` on `GridView` since it isn't a `Widget`, but is there some simple way of wrapping a GridView inside a widget that I'm missing? I'm coming from Kivy, and in that library all "views" are also widgets.

**Comment by HiPeople21** [comment]
I'm not sure if it would work as the CSS branch isn't working for me right now, but I believe you should be able to call `animate` on `GridView` since `GridView` is a subclass of `Screen` which is a subclass of `Widget`.

**Reply by unwize** [reply]
I've been attempting to animate `GridView`, but it simply won't work. Do you have an example snippet I could look at?

**Reply by HiPeople21** [reply]
Sorry I don’t. As I said, the branch wasn’t working for me so I was just saying that it might work because of the inheritance.
